src/roman_datamodels/validate.py

```diff
@@ -54,14 +54,17 @@ def _check_type(validator, types, instance, schema):
 validator_callbacks.update({"type": _check_type})
 
 
-def _check_value(value):
+def _check_value(value, schema_uri=None):
```

Comment:
The `schema_uri` argument is not used and can be removed.

Reply:
Thanks for the catch! I removed the unused argument.